src/assets/i18n/en-US.json

```diff
@@ -210,6 +210,72 @@
     "wishlist": {
       "emptyListMessage": "Sorry, your wishlist is empty!",
       "additionalMessage": "Add product to your wishlist"
+    },
+    "operations": "Operations",
+    "emptyList": "Sorry, your wishlist is empty."
+  },
+  "registrationPage": {
+    "form": {
+      "register": "Register",
+      "firstName": "First name",
+      "lastName": "Last name",
+      "email": "E-mail",
+      "password": "Password",
+      "confirmPassword": "Confirm password",
+      "termsAndConditions": "I have read and agree with the Terms & Conditions",
+      "signUp": "Sign up",
+      "registrationSuccess": "Registration successful",
+      "alreadyRegistered": "Already registered?",
+      "loginHere": "Login here"
+    },
+    "placeholders": {
+      "firstName": "Enter first name",
+      "lastName": "Enter last name",
+      "email": "Enter e-mail",
+      "password": "Enter password",
+      "confirmPassword": "Confirm password"
+    },
+    "messages": {
+      "minLength": "Minimum required characters: ",
+      "maxLength": "Maximum allowed characters: ",
+      "required": "This field is required",
+      "wrongFormat": "Wrong format",
+      "alreadyTaken": "This e-mail is already taken",
+      "checkingEmail": "Checking e-mail...",
+      "passwordMatch": "Passwords must be the same"
     }
+  },
+  "loginPage": {
+    "form": {
+      "login": "Login",
+      "email": "E-mail",
+      "password": "Password",
+      "rememberMe": "Remember me",
+      "loginSuccess": "Login successful",
+      "loginError": "Login unsuccessful",
+      "logoutSuccess": "Logout successful",
+      "logoutError": "Logout unsuccessful",
+      "notRegistered": "Not registered?",
+      "registerHere": "Register here",
+      "userNotFound": "User not found",
+      "incorrectPassword": "Incorrect password",
+      "anErrorOccured": "An error occured"
+    },
+    "placeholders": {
+      "email": "Enter e-mail",
+      "password": "Enter password"
+    },
+    "messages": {
+      "minLength": "Minimum required characters: ",
+      "maxLength": "Maximum allowed characters: ",
```

Comment:
let's use params for it

Reply:
and for all where we have this logic

Reply:
I don't understand this. I can not put parameters inside JSON file. Am I misunderstanding something?